test/e2e/xsite/xsite_test.go

```diff
@@ -231,6 +231,16 @@ func TestDefaultTLSLoadBalancerWithPort(t *testing.T) {
 	testCrossSiteView(t, true, ispnv1.CrossSiteSchemeTypeOpenShift, ispnv1.CrossSiteExposeTypeLoadBalancer, 1443, 1, DefaultTLS, nil)
 }
 
+func TestCrossSiteViewOpenshiftRoute(t *testing.T) {
+	tutils.SkipForExposeType(t, ispnv1.ExposeTypeNodePort, ispnv1.ExposeTypeLoadBalancer)
```

Comment:
Maybe skipping based on whether `Route` API is available would be more consistent with the rest of the test rather then on test suite configuration? (We are not skipping LoadBalancer tests just because NodePort expose type is configured)

Reply:
any pointers how to do that?

Reply:
We can make use of the `IsGroupVersionSupported` method like [here](https://github.com/infinispan/infinispan-operator/blob/main/test/e2e/utils/common.go#L216) to see if it's supported and if not call skip on the test.